src/mixins/wallet.js

```diff
@@ -72,7 +73,9 @@ export default {
         'connected_wallet',
         1
       );
-      return this.initWallet(connection);
+      return isSkippedLogin
```

Comment:
Does this flag means `Login was skipped` or it means we want to skip login when calling this function?
If it is latter, we should call this flag `skipLogin`/`isSkipLogin`

Reply:
How about `shouldSkipLogin`?

Reply:
Yes it would be nice